src/store/index.ts

```diff
@@ -70,9 +70,17 @@ export const store = {
             return settings;
         },
 
-        // TODO: Add reset actions for other modules
         async resetProfileModule({dispatch}: Context) {
             return await dispatch('profile/reset');
+        },
+        async resetProfilesModule({dispatch}: Context) {
```

Comment:
I don't see much value in having these wrappers that just dispatch another action here. Why I first suggested having the reset function in the index file was that we'd have a *single* reset method that calls all the reset methods of submodules (i.e. caller doesn't have to care what the reset does and call each one of these separately).

Reply:
To reword my earlier comment:

- If it's possible to have *one* action on the index store, that calls reset on the substores, then that's the way I'd like this implemented
- If we need to call different resets on different components (as in GameSelectionScreen.vue and Profiles.vue in this PR), then it makes no sense route the calls through the main store. Instead, call the reset methods directly from the substores
- If none of the resets need to be async (i.e. they just call the commit), then we should yeet the wrapping dispatch methods and call the commits directly
  - However, for consistency's sake it's nice for each substore to implement an identical method, so if one actually needs the async call, then use actions/dispatch for all
  - Also if you find it reasonable to assume that one of the resets will need to do something async soon, then it's okay to future proof this. But I personally wouldn't do it if it's not needed.